config-overrides.js

```diff
@@ -25,11 +25,11 @@ const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
 
```

Comment:
Please git pull from develop branch and discard these changes. I got it working in develop branch.